### ISSUE TITLE
oxAuth fail to prepare JMS 

### ISSUE THREAD

**Issue by mzico** [GluuFederation/oxAuth#1391]
Enabling "enabledOAuthAuditLogging" + JMS configurations throwing NPE. 
Log snippet: 

```
2020-06-09 23:38:16,561 ERROR [8A-0C-45-FB-C8-BB] [Thread-22012] [org.gluu.oxauth.audit.ApplicationAuditLogger] (ApplicationAuditLogger.java:167) - Can't send message, please check your activeMQ configuration.
java.lang.NullPointerException: null
    at org.gluu.oxauth.audit.ApplicationAuditLogger.loggingThroughJMS(ApplicationAuditLogger.java:153) [classes/:?]
    at org.gluu.oxauth.audit.ApplicationAuditLogger.sendMessage(ApplicationAuditLogger.java:85) [classes/:?]
    at org.gluu.oxauth.audit.ApplicationAuditLogger$Proxy$_$_WeldSubclass.sendMessage$super(Unknown Source) [classes/:?]
    at sun.reflect.GeneratedMethodAccessor230.invoke(Unknown Source) ~[?:?]
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_212]
    at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_212]
    at org.jboss.weld.interceptor.proxy.TerminalAroundInvokeInvocationContext.proceedInternal(TerminalAroundInvokeInvocationContext.java:51) [weld-core-impl-3.1.2.Final.jar:3.1.2.Final]
    at org.jboss.weld.interceptor.proxy.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:78) [weld-core-impl-3.1.2.Final.jar:3.1.2.Final]
    at org.gluu.service.cdi.async.AsynchronousInterceptor$1.get(AsynchronousInterceptor.java:36) [oxcore-service-4.1.1.Final.jar:?]
    at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1590) [?:1.8.0_212]
    at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```

Reference ticket: https://support.gluu.org/maintenance/8477/oxauth_audit-logs-filled-with-activemq-configuration-error

**Comment by nynymike** [comment]
Fix will make it into 4.2

**Comment by yurem** [comment]
Fixed
